predicates: []

default_goal: ['(AT TARGET PLACE_LOC)']

